src/test/scala/org/scalameter/MeasurerTest.scala

```diff
@@ -0,0 +1,20 @@
+package org.scalameter
+
+import org.scalatest.{FunSuite, Matchers}
+import org.scalameter.execution.JvmRunner
+
+
+abstract class MeasurerTest[M <: Measurer] extends FunSuite with Matchers {
```

Comment:
Could you additionally add a few integration-like tests in the spirit of `JavaPerformanceTestTest.scala`?